src/main/java/com/epam/reportportal/extension/azure/AzureExtension.java

```diff
@@ -184,4 +224,165 @@ private void removeListeners() {
 		pluginCommandMapping.put("testConnection", new TestConnectionCommand());
 		return pluginCommandMapping;
 	}
+
+	@Override
+    // Never called method. Connection is tested via the command.
+	public boolean testConnection(Integration system) {
+		return false;
+	}
+
+	@Override
+    // TODO: Implement in the future
+	public Optional<Ticket> getTicket(String id, Integration system) {
+		return Optional.empty();
+	}
+
+    @Override
+    // TODO: Implement in the future
+    public Ticket submitTicket(PostTicketRQ ticketRQ, Integration system) {
+        return null;
+    }
+
+	@Override
+	public List<PostFormField> getTicketFields(String issueType, Integration integration) {
```

Comment:
Please split such looooong functions in smaller ones. I can't see more than 40 lines on github screen.

Reply:
Decomposed this method into several smaller